frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -182,7 +181,7 @@ interface SessionCompletionProp {
 }
 
 const SessionBox = styled.div`
-  padding: 40px;
+  padding: 1.5vh 1.5vw;
```

Comment:
My thoughts on the session box:
- may need more padding
- maybe make the image smaller
- make the font size smaller
- probably want to do justify-content: flex-end; I'm getting a weird thing where "session 2" text is positioned higher than the rest of the texts 

Not necessarily something you need to fix in this PR, but just noting it for a bug ticket or something

Reply:
Good catch! I am working to fix these (and I am pretty sure I introduced this bug, so I am fixing it :)